### PR TITLE
Ref to getter function in docs/introduction-to-smart-contracts.rst

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by jamesray1** [comment]
@chriseth

**Comment by chriseth** [comment]
Sorry,  I did not realize you cannot see the test failure:
```
/home/travis/build/ethereum/solidity/docs/introduction-to-smart-contracts.rst:126: WARNING: undefined label: _getter_functions (if the link has no caption the label must precede a section header)
```

Please run `make html` in the   `docs` directory to run the tests.

**Comment by jamesray1** [comment]
This needs #2658 to be merged first.

File: docs/introduction-to-smart-contracts.rst [hunk 1]
```diff
@@ -133,7 +133,7 @@ too far, though, as it is neither possible to obtain a list of all keys of
 a mapping, nor a list of all values. So either keep in mind (or
 better, keep a list or use a more advanced data type) what you
 added to the mapping or use it in a context where this is not needed,
-like this one. The getter function created by the ``public`` keyword
+like this one. The :ref:`getter-functions` created by the ``public`` keyword
```
Comment (chriseth) [inline docs/introduction-to-smart-contracts.rst]: I would say this should still read `getter functions` (with space).

Comment (jamesray1) [reply docs/introduction-to-smart-contracts.rst]: I get what you mean, it should still say getter function, but I think the edit would make it say "getter functions".

**Review by chriseth — APPROVED** [review]

**Comment by jamesray1** [comment]
@chriseth, the Travis CI build passed.

**Pull request merged**